application/tests/_ci_phpunit_test/CIPHPUnitTestCase.php

```diff
@@ -119,12 +119,13 @@ public function newModel($classname)
 	 * Create a library instance
 	 *
 	 * @param string $classname
+	 * @param mixed  $args
```

Comment:
This shoulb be `array` according to CodeIgniter's doc comment.
Would you please fix it?

Reply:
Fixed now.